spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/interpreter/stages/TypeParser.scala

```diff
@@ -22,23 +22,22 @@ import java.util.regex.Pattern
 
 import org.apache.spark.sql.Column
 import org.apache.spark.sql.expressions.UserDefinedFunction
-import org.apache.spark.sql.functions._
+import org.apache.spark.sql.functions.{lit, _}
```

Comment:
Does the `lit` need to be specified likethis?

Reply:
Fixed, thanks